contact.html

```diff
@@ -4,8 +4,37 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Contact</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web&display=swap" rel="stylesheet">
 </head>
 <body>
+    <address>
+        <h1>Say hello!</h1>
+        <img src="https://media.giphy.com/media/3oz8xTAJIQD6JWfTUc/giphy.gif" alt="wave">
+        <p>By Phone: 972-123-4567</p>
+        <form action="mailto:jrios1991@gmail.com" method="POST">
+            <p>Or by Email</p>
+            <label for="fname">First Name
+                <input type="text">
+            </label>
+            <br>
+            <label for="lname">Last Name
+                <input type="text">
+            </label>
+            <br>
+            <label for="phone">Phone Number
+                <input type="text">
+            </label>
+            <br>
+            <label for="email">Your Email
+                <input type="email">
+            </label>
+            <br>
+            <button type="submit">Contact Me!</button>          
+        </form>
+        <a href="index.html">Home</a>
+    </address>
     
```

Comment:
You followed well a standard of including NO blank lines, up to this point (and this ones very minor).  ;0)
It's good practice to avoid blank lines, unless a section of code is confusing without the blank line.  And then, just a single blank line.  Nicely maintained throughout this file.